src/main/java/com/qunite/api/service/QueueServiceImpl.java

```diff
@@ -51,12 +53,43 @@ public Optional<Integer> getMemberPositionInQueue(Long memberId, Long queueId) {
         .map(entry -> entry.getEntryIndex() + 1);
   }
 
+  @Override
+  @Transactional(isolation = Isolation.REPEATABLE_READ)
+  public void changeMemberPositionInQueue(Long memberId, Long queueId,
+                                          Integer newIndex, String username) {
+    if (isUserQueueCreatorOrManagerByCredentials(queueId, username)) {
+      entryRepository.findById(new EntryId(memberId, queueId)).ifPresent(entry -> {
+        var currentIndex = entry.getEntryIndex();
+        if (!currentIndex.equals(newIndex)) {
+          var startIndex = Math.min(currentIndex, newIndex);
+          var endIndex = Math.max(currentIndex, newIndex);
+          var increment = currentIndex < newIndex ? -1 : 1;
```

Comment:
I really don't understand why are you using `var` even with such easy types like `int`. I am supposed to read the line of code to catch why increment is `var`, but not an `int`